meteor-app/imports/ui/member/add/member-add-container.js

```diff
@@ -1,18 +1,18 @@
 import { Meteor } from 'meteor/meteor'
 import { withTracker } from 'meteor/react-meteor-data';
-import MemberAdd from '/imports/ui/member/member-add'
+import MemberAdd from './member-add'
```

Comment:
I'm 50/50 about this. I definitely don't like '../../thing/file' because it means you need to know where the current file is to work out the relative location.
If it's in the same directory, this assumption is perhaps ok, but if you want to move things around, this is harder to track